src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/vr_is_ready_request_test.cc

```diff
@@ -127,23 +127,36 @@ class VRIsReadyRequestTest
     event.set_smart_object(*msg);
   }
 
+  void HMICapabilitiesExpectations() {
+    std::vector<hmi_apis::FunctionID::eType> interfaces_to_update{
+        hmi_apis::FunctionID::VR_GetLanguage,
+        hmi_apis::FunctionID::VR_GetSupportedLanguages,
+        hmi_apis::FunctionID::VR_GetCapabilities};
+    EXPECT_CALL(mock_hmi_capabilities_, GetInterfacesToUpdate())
+        .WillOnce(Return(interfaces_to_update))
```

Comment:
@IGapchuk I would recommend to use such approach: 
```cpp
EXPECT_CALL(mock_hmi_capabilities_, GetInterfacesToUpdate())
.Times(2)
.WillRepeatedly(Return(interfaces_to_update))
```

Reply:
@AByzhynar now it is redundant because in the new implementation we have only one function call. So, I have changed the UT - https://github.com/smartdevicelink/sdl_core/pull/3265/files#diff-21f90c17ded64c44e77a62a090ebd49fR135